sync/fromDisk/assessments.js

```diff
@@ -76,6 +76,7 @@ function getParamsForAssessment(assessmentInfoFile, questionIds) {
     student_group_create: !!assessment.studentGroupCreate,
     student_group_join: !!assessment.studentGroupJoin,
     student_group_leave: !!assessment.studentGroupLeave,
+    help_link: assessment.helpLink || null,
```

Comment:
To go along with the change to `not null` for this column:

```suggestion
    help_link: assessment.helpLink || false,
```

Reply:
Just checking my understanding, I thought this line was paired with the `assessment` table's` help_link`, which should be text, but this makes it a boolean, right? Changing the `help_link` column in `pl_courses.pg` was fine and makes sense, but adding this change starts breaking things (Clicking on the button opens the tab with `Error: Not Found` with additional data saying `"url": "/pl/course_instance/1/instance_question/1/true"` ) so just want to make sure before looking into other things I might need to fix.

Reply:
Sorry, this was just a bad suggestion in my part! I got my wires crossed with the two identically-named columns. Consider renaming the one on the courses table to `help_link_enabled` yo reduce confusion.

Reply:
Oh yes, that renaming will make things much easier, I'll go do that, thanks!